fireflower/targets.py

```diff
@@ -95,30 +95,63 @@ def __init__(self, path, compressed=True, kwargs_in=None, kwargs_out=None,
         self.kwargs_out = kwargs_out
         super(S3CSVTarget, self).__init__(path, format)
 
-    @contextmanager
-    def _open_writer(self):
+    @staticmethod
+    def write_csv_rows(csv_writer, values, header=None):
```

Comment:
Do we need this method? Lots of short methods can make code hard to read. Also, the method name is really similar to write_csv_tuples, which is a little confusing. It might make sense to inline the method. Up to you.